packages/react/src/use-logto.ts

```diff
@@ -0,0 +1,21 @@
+import { useContext } from 'react';
+
+import { LogtoContext, LogtoContextProperties } from './context';
+
+export default function useLogto() {
+  const context = useContext<LogtoContextProperties>(LogtoContext);
```

Comment:
nit: no need to specify the return types here. useContext should recognize the return type automatically.